config/locales/user_mailer/en.yml

```diff
@@ -11,26 +11,34 @@ en:
       'yes': 'yes'
     email_changed:
       help: >
-        If you did not change your email address, please contact the %{app}
-        Customer Contact Center at %{link}.
+        If you did not change your email address, please immediately call the
+        %{app} customer contact center at 1-844-USA-GOV1 (1-844-872-4681)
```

Comment:
should we hard code `1-844-USA-GOV1 (1-844-872-4681)` as a constant somewhere and then pass it into these translations? Seems like it would be easy to mess it up when typed in multiple locations